reporting/api/src/main/java/org/jboss/windup/reporting/model/ApplicationReportModel.java

```diff
@@ -66,27 +65,13 @@
     Boolean getDisplayInGlobalApplicationIndex();
 
     /**
-     * Indicates whether or not to display this in the list of all applications. Usually this would be true for a main "overview" type report for a
-     * particular application, and false for everything else.
-     */
-    @Property(DISPLAY_IN_APPLICATION_LIST)
-    Boolean getDisplayInApplicationList();
-
-    /**
-     * Indicates whether or not to display this in the list of all applications. Usually this would be true for a main "overview" type report for a
-     * particular application, and false for everything else.
-     */
-    @Property(DISPLAY_IN_APPLICATION_LIST)
-    void setDisplayInApplicationList(Boolean displayInApplicationList);
-
-    /**
-     * Indicates whether to display this report in the navigation index for the current application.
```

Comment:
I think we have too much flags and this one basically maps the value sfrom Main_report flag.